Add .github for CODEOWNERS

seemethere opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#61598 Add .github for CODEOWNERS**

I'd like to be notified on changes to the github actions workflows, add
this so I can be notified.

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D29685783](https://our.internmc.facebook.com/intern/diff/D29685783)

seemethere committed:
Add .github for CODEOWNERS

I'd like to be notified on changes to the github actions workflows, add
this so I can be notified.

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

seemethere commented:
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29685783).

walterddr commented on CODEOWNERS:
```diff
@@ -38,3 +38,8 @@
+# Github Actions
+# This list is for people wanting to be notified every time there's a change
+# related to Github Actions
+/.github/ @seemethere
```
should we add others from the team?

seemethere replied:
Yup, If you'd like to be included on this list please comment below, I'm planning on merging this on Thursday July 15th to give other people time to comment here

janeyx99 replied:
me

zhouzhuojie replied:
me pls

driazati replied:
i nominate myself

seemethere committed:
Update on "Add .github for CODEOWNERS"


I'd like to be notified on changes to the github actions workflows, add
this so I can be notified.

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D29685783](https://our.internmc.facebook.com/intern/diff/D29685783)

[ghstack-poisoned]

seemethere commented:
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29685783).
